### PR TITLE
update rails version

### PR DESCRIPTION
**Pull request by lucianosousa**

### CODE REVIEW

**Comment by tauil** [comment]
Where is the Gemfile? There's only the lock.

**Comment by lucianosousa** [comment]
Rails is not locked there.
Minor versions are updated without any change.

**Comment by tauil** [comment]
You mean that the Gemfile.lock has changed based on Gemfile that currently is in master?

**Comment by lucianosousa** [comment]
yes

File: Gemfile.lock [hunk 1]
```diff
@@ -423,3 +423,6 @@ DEPENDENCIES
   unicorn
   validates_timeliness (~> 3.0.14)
   will_paginate
+
+BUNDLED WITH
+   1.10.6
```
Comment (tauil) [inline Gemfile.lock]: Just tested and those lines have been removed from mine after runnning bundle from this branch. Any idea?

**Comment by lucianosousa** [comment]
Ignore it

**Pull request merged**